### PR TITLE
Add support for YAML CloudFormation templates

### PR DESCRIPTION
**Pull request by cb372**
If the template file has a `.yml` or `.yaml` extension, it is assumed to be in YAML format and RiffRaff will convert it to JSON on the fly. Otherwise it is assumed to be in JSON format and will be used as-is.

### CODE REVIEW

File: magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala [hunk 1]
```diff
@@ -15,6 +18,26 @@ object UpdateCloudFormationTask {
...
 }
 
+object JsonConverter {
+
+  /**
+    * Return the template's content as JSON,
+    * converting it from YAML if necessary.
+    */
+  def convert(template: Path): String = template.extension match {
+    case Some("yml") | Some("yaml") =>
```
Comment (adamnfish) [inline magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala]: I didn't know that `|` syntax! Thank you!

**Comment by jamespamplin** [comment]
:tada: nice one!

**Comment by davidfurey** [comment]
Works in CODE.

**Comment by cb372** [comment]
@davidfurey thanks for testing!

**Pull request merged**